akka-docs/rst/scala/code/docs/http/scaladsl/server/directives/JsonStreamingExamplesSpec.scala

```diff
@@ -132,10 +132,9 @@ class JsonStreamingExamplesSpec extends RoutingSpec {
 
     Get("/tweets").withHeaders(AcceptCsv) ~> route ~> check {
       responseAs[String] shouldEqual
-        """|1,#Akka rocks!
-           |2,Streaming is so hot right now!
-           |3,You cannot enter the same river twice."""
-        .stripMargin
+        "1,#Akka rocks!" + "\n" +
```

Comment:
I see. It's example code.
```scala
val sep = ByteString(sys.props("line.separator"))
```
is an alternative but your fix should be fine as well.